docs/guides/HttpRequestTypes.md

```diff
@@ -1,6 +1,6 @@
 ## HTTP: Request Types
 
-The [basic guide to sending requests](/docs/guides/HttpSendRequestReceiveResponseHandleFailure.md)
+The [basic guide to sending requests](https://github.com/Workiva/w_transport/blob/master/docs/guides/HttpSendRequestReceiveResponseHandleFailure.md)
```

Comment:
I'm okay with this because I don't see these guides changing any time soon, but this is another unfortunate limitation of having to use absolute URLs: by pointing to `master`, we will break historical linking. Let's say we release a v4 in the future but someone wants to browse the repo at the latest v3 revision - these links would be broken because they don't allow the user's current context to be taken into account.